Barrykilby/charity-donation
# App development

Created a Checkout UI Extension for RNIB.

To Test:
1. Go to settings
2. Checkout
3. Customize Checkout
4. You will see the Checkout block on the left side.

[Jira Ticket](https://space48.atlassian.net/browse/RNIB-202)
